src/generate_wrapper.py

```diff
@@ -639,11 +638,8 @@ def process_docstring(f):
         for param in f["parameters"]:
             param_type = adapt_param_type(param["type"])
             # remove const and &
-            param_type = param_type.replace("const", "")
             param_type = param_type.replace("Standard_Boolean &", "bool")
```

Comment:
@lalvarezt You missed this case, it should also be moved to fix_type otherwise all bool return types are transformed to bool &. This might have strong side effects.